ui/app/components/clients/page/acme.hbs

```diff
@@ -27,6 +27,7 @@
         @label="Total ACME clients"
         @subText="The total number of ACME requests made to Vault during this time period."
         @value={{this.totalUsageCounts.acme_clients}}
+        @tooltipText="This number is the total for the queried date range. The chart displays a monthly breakdown of total clients per month."
```

Comment:
This tooltip was added to the overview. Adding to every total stat number because there has been customer confusion on what this number means.
<img width="1076" alt="Screenshot 2024-04-22 at 12 04 17 PM" src="https://github.com/hashicorp/vault/assets/68122737/9bbffb25-8268-4252-8036-7cb39835613e">